### ISSUE TITLE
Codex: Blood Angels (WIP)

### ISSUE THREAD

**Issue by Kangodo** [BSData/wh40k-7th-edition#598]
Today I will start on the BA-codex, and I hope to have everything finished by tomorrow.
The first feedback could go in this thread as to avoid BA-spam for the first few days.

**Comment by senpai514** [comment]
You got the formation in today's white dwarf?

**Comment by Kangodo** [comment]
No, not yet. I only have the stuff from the Codex.
That is all done (except for some cleaning up and the errors I know I made but cannot find).

Looking into that one right now.
Edit: Added it in and fixed some bugs.
I still think the Formations look ugly, any idea what causes that and how I can fix it?

Another issue is the default selection for weapons and, like any codex, the way the Wargear-list works.

**Comment by senpai514** [comment]
what do you mean by ugly?
for the weapons and such are you referring to squad over characters?
also double check the storm eagle I believe IA2.2 only has that for space marines, dark angels and wolves.

**Comment by Kangodo** [comment]
Well, when I add the Flesh Tearers formation I still have to add Tactical Squads till I fulfil the minimum.
How do I make it so that it automatically adds three Tactical Squads, because it didn't seem to work with three links nor with one link with a modifier.

For melee weapons I have 5 choices and then a 'standard' box for BP, BG or Chainsword (same with Ranged).
I'm reading the entire codex to find out if I could just make it 8 choices, but I have to watch out for illegal combinations.

I'm not sure on what to do with the Storm Eagle, FW mailed us saying they would add them to BA in a FAQ but we can't wait forever.

**Comment by senpai514** [comment]
I'll take a look at the formation for you.
Ok the formation is tweaked now and puts out 3 squads. I find that making and entry with a min/max and then putting the squad in that entry is the best way to auto fill formation quantities.

yeah FW does take a while to put out simple updates maybe just hide it for temp until they finally do as they said. Sucks, I know but only so much we can do ourselves about legal not legal models and FAQ's

**Comment by Kangodo** [comment]
Aah, thank you!
But the squad inside still had modifiers, so you needed 9 now ;) I fixed that.
Gunship is hidden till we get a decent FAQ, the e-mail is not enough to keep it in.

Unless I get some more feedback in the next hour, I will do a release and let people play with it.

**Comment by senpai514** [comment]
Ok sounds good I have a big update ready for release for my catalogue.  I'm
waiting on a downloadable copy of the codex for myself so I can take a look
through it.

**Comment by senpai514** [comment]
You should do the same to the battle company formation as well

**Comment by Kangodo** [comment]
The ePub was released yesterday, I found a link on /tg/ 
It's a lot easier to work than with the Codex on my lap.

**Comment by senpai514** [comment]
Can you send link to thommy.abbott@gmail.com please

**Comment by senpai514** [comment]
FLESH TEARERS BLOOD RAIN STRIKE FORCE

Formation
1 Death Company Squad
1 Vanguard Veteran Squad
1 Assault Squad
1 Furioso Dreadnought
1 Stormraven Gunship

Restrictions
All models in this Formation must either be equipped with jump packs or begin the game embarked upon the Formations Stormraven Gunship.

Special Rules:
Crusader
Crimson Thunderbolt: On a turn in which a unit from this Formation disembarks from this Formation's 
Stormraven Gunship, it can re-roll failed charge roll and has the Rage special rule.
Poised to Strike: All units in this Formation equipped with jump packs must be placed in Deep Strike 
Reserve. When making Reserve Rolls, make a single roll for the entire Formation, which you can choose to re-roll. If successful, all units in this Formation arrive from Reserve.
Rain of Blood: When a unit from this Formation arrives by Deep Strike, it gains the Counter-attack and Fearless special rules until the start of your next turn.

**Comment by senpai514** [comment]
ANGEL'S FURY SPEARHEAD FORCE

Formation
3 Tactical Squads
3 Stormraven Gunships

Restrictions
Each Tactical squad must include 10 models. The Sergeant of each Tactical squad in this Formation must take a teleport homer as a free upgrade. This Formation's Tactical squads cannot take Dedicated Transports, and must begin the game embarked in the Stormraven Gunships from this Formation.

Special Rules
Augur Triangulation: If a friendly unit with the Blood Angels Faction arrives from Deep Strike Reserve within 12" of at least two models from this Formation equipped with teleport homers, then it does not scatter, and can charge on the same turn it arrives.
Objective Secured: All Troops units from this Formation have the Objective Secured special rule.
Spearhead Strike Force: When making Reserve Rolls, make a single roll for the entire Formation, which you can choose to re-roll. If successful, all units arrive from the Formation. You can make a Reserve Roll for this Formation from the start of Turn 1.

**Comment by ghost** [comment]
Any suggestion for what happens to Malakim Phoros' option to take an Honour Guard?

**Comment by Kangodo** [comment]
Thank you Senpai, I know about those but it looks rather blurry.
I will start to add stuff in the weekend when both the WD and the book are released.

As for Malakim, I will remove the option to take Honour Guard.
Honour Guard can't be taken in the Codex and we have to wait for a FQ-FAQ.

**Comment by senpai514** [comment]
yeah I found some clearer images for accuracy figured I copy them verbatim
so you could get a head start on these

**Comment by Kangodo** [comment]
Yes, I found them too.
BoLS has them up now, so will add them tonight.

**Comment by Kangodo** [comment]
We will get a Detachment that can only contain Terminators, Veterans or Furioso in the Elite-slot.
Does anyone have any idea on how to handle this?

The easiest option is probably to make it a Formation, but should I do that?

**Comment by amis92** [comment]
I don't know really. Making it a formation is against the format and all that. I mean, people still have a problem with different detachments and making it even more complicated by pushing special Detachment as Formation, well...

Is there so many other Elites they can't get a modifier saying "if parent's force type is That-Detachment, set maxInForce to 0"? Or is it rather a question of how maintainable it will be in future? :)

**Comment by Kangodo** [comment]
We usually have 14 Elites and 16 (13 if you count Captain and Terminator Captain as one) HQ's.
In the Detachment we have:
Sternguard/Vanguard/Terminator/Terminator Assault (max 10 of this combination)
Furioso (max 6)

And for HQ:
Captain (or Karlaen), Chaplain or Librarian (they must have Terminator armour)

So I could make modifier that hides them if this specific Detachment is chosen.
But then I'd still have the issue with the maximum of 10 Vets/Termies and 6 Furioso.

**Comment by ghost** [comment]
You could create a Shared Entry (min=1, max=1, maxinforce=-1) with a modifier setting maxinforce=10 if parent force is instance of _detachment name_. Then link the Vets/Termies to this shared entry. Give the Furioso a modifier setting maxinforce=6 if as above. Hide the ineligible units the same way, and give the shared entry an innocuous name.

**Comment by Kangodo** [comment]
Most of that I could do, but I'm not quite sure what you mean with the shared entry for Vets and Terminators.
Are you talking about an entry called Veterans, which gets a maxinforce of 10 and is only visible in this Detachment?
Under that Veteran-entry I could link all units in an entry-group with a max-selection of 1.
That way people could add 10 Veteran-entries and each entry can be either one of those 4.
Or are you talking about something else?

For now I made them a Formation, just for people that plan to play games during the holidays.

Another thing:
I made an entry called 'Detachment-source' which allows people to select the source of the Detachment.
Relics and Warlord Traits are hidden and shown depending on what source they selected.
I personally like this, as it allows people to check out other Traits and Relics without loading a new roster.
The downside is that it doesn't give an error-message when you make illegal options, like an Archangel-CAD.
Opinions?

**Comment by amis92** [comment]
Sorry, I really am: Sh*t, it's Emperor damn it complicated. I can't help you. :[

**Comment by ghost** [comment]
@Kangodo - I was thinking of a Shared Entry called something like "elite unit". It wouldn't contain anything other than the modifier. You would make the 4 veteran/termie squads in the usual way & then link them to the shared entry, same as if you were linking them to a dedicated transport option.
So when someone tried to build this special detachment, selecting 6 sternguard squads & 6 vanguard squads for example, they would get an error saying "You have too many elite units". 
Someone making a CAD would also see the "elite unit" entry in their roster, but that doesn't matter, because it wouldn't do anything to them.

**Comment by Climer00** [comment]
the Furiouso Dred does not have the option to put a frag canon for +5 pts. Please edit when you get a chance and if you have already fixed thanks. And thank you for putting all this together to start with.

**Comment by Kangodo** [comment]
You're welcome!
The Furioso Dread does have that option, it's in a submenu.

I also had an idea for the Detachment that might work better.

> -------Set MaxinForce to 10 (6 for Furioso) 
> -----------if Force has equal to 1 Selection of "Archangels Strike Force". 
> -------Decrement MaxinForce by 1 (repeat) "1" time for every "1" selection in "Parent Force" of "Assault Terminators" 
> -----------if Force has equal to 1 Selection of "Archangels Strike Force". 
> -------Decrement MaxinForce by 1 (repeat) "1" time for every "1" selection in "Parent Force" of "Sternguard Veterans" 
> -----------if Force has equal to 1 Selection of "Archangels Strike Force". 
> -------Decrement MaxinForce by 1 (repeat) "1" time for every "1" selection in "Parent Force" of "Vanguard Veterans" 
> -----------if Force has equal to 1 Selection of "Archangels Strike Force". 

That might work better, though it's a lot more work for me.

**Comment by Flashpoint-** [comment]
Hi Kangodo / guys and gals
Blood Angels have formations in White Dwarf #47 that I can't find in the 
battlescribe 1.15.04
Blood Angels: Codex (2014) v4
1. ANGEL'S WRATH INTERVENTION FORCE
2. ANGEL'S FURY SPEARHEAD FORCE
3. FLESH TEARS BLOOD RAIN STRIKE FORCE
Thank you happy 2015

**Comment by Kangodo** [comment]
Thank you, happy 2015 too!
These are in the current file, not 100% sure if they were also in v4.
But I will make some changes tomorrow and upload v5.
